Directory.Packages.props

```diff
@@ -33,7 +33,22 @@
     <PackageVersion Include="Microsoft.Azure.SignalR" Version="1.25.1" />
     <PackageVersion Include="Microsoft.Extensions.Azure" Version="1.7.2" />
     <!-- Azure Management SDK for .NET dependencies -->
-    <PackageVersion Include="Azure.Provisioning" Version="0.1.0-beta.2" />
+    <PackageVersion Include="Azure.Provisioning" Version="0.2.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.AppConfiguration" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.ApplicationInsights" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.CognitiveServices" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.CosmosDB" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.EventHubs" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.KeyVault" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.OperationalInsights" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.PostgreSql" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.Redis" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.Resources" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.Search" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.ServiceBus" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.SignalR" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.Sql" Version="0.1.0-beta.1" />
+    <PackageVersion Include="Azure.Provisioning.Storage" Version="0.1.0-beta.1" />
```

Comment:
Why not use a variable here?

Reply:
Because they will eventually start releasing at a different cadence. Right now these package are all in the sdk/provisioning folder but they will end up splitting them out into sdk/[service] folders and will tend to release on a cadence tied to the underlying management plane packages.

Reply:
Can't we change it when that happens?